docs/reference/overview_of_strategies.rst

```diff
@@ -1014,17 +1014,17 @@ following four-vectors :math:`(P(C \,|\, CC), P(C \,|\, CD), P(C \,|\, DC), P(C
 +------------------+-----------------------+--------------------------+
 | Name             | Four-vector           | Axelrod Library Name     |
 +==================+=======================+==========================+
-| `ZD-Extort-2`_   | (8/9, 1/2, 1/3, 0)    | :code:`ZDExtort2`        |
+| ZD-Extort-2      | (8/9, 1/2, 1/3, 0)    | :code:`ZDExtort2`        |
 +------------------+-----------------------+--------------------------+
-| `ZD-Extort-2v2`_ | (7/8, 7/16, 3/8, 0)   | :code:`ZDExtort2v2`      |
+| ZD-Extort-2v2    | (7/8, 7/16, 3/8, 0)   | :code:`ZDExtort2v2`      |
 +------------------+-----------------------+--------------------------+
-| `ZD-Extort-4`_   | (11/17, 0, 8/17, 0)   | :code:`ZDExtort4`        |
+| ZD-Extort-4      | (11/17, 0, 8/17, 0)   | :code:`ZDExtort4`        |
 +------------------+-----------------------+--------------------------+
-| `ZD-Gen-2`_      | (1, 9/16, 1/2, 1/8)   | :code:`ZDGen2`           |
+| ZD-Gen-2         | (1, 9/16, 1/2, 1/8)   | :code:`ZDGen2`           |
 +------------------+-----------------------+--------------------------+
-| `ZD-GTFT-2`_     | (1, 1/8, 1, 1/4)      | :code:`ZDGTFT2`          |
+| ZD-GTFT-2        | (1, 1/8, 1, 1/4)      | :code:`ZDGTFT2`          |
```

Comment:
> It's an internal link on the page to the relevant section

I can't _find_ a relevant section though - can you point me at where you expect these 6 links to go?

> On c0f0a4a you say there are code links but they don't actually link anywhere

To rephrase, I said I thought they _might_ be supposed to be code links, but that didn't quite make sense because the third column had code links

So looking at the current rendering of https://axelrod.readthedocs.org/en/latest/reference/overview_of_strategies.html#zero-determinant-srategies it's:

![screen shot 2015-11-24 at 13 14 48](https://cloud.githubusercontent.com/assets/817118/11367730/85df23a0-92ad-11e5-9ad5-09b9746b1699.png)

and you can see that the first column is a failed rendering.

I couldn't tell what was intended, because there's no obvious section for them to link to, and they're not code links because that's what's in the third column, so in the absence of having any idea of what they _should_ link to, I just made them not be links.

If you can help me see where they're _supposed_ to go, I'll gladly change that!